### ISSUE TITLE
Gloom broken using lazy pack generator and "Add Black-screen fix (Xbox UI)"

### ISSUE THREAD

**Issue by mark007** [HolographicWings/TOTK-Mods-collection#141]
I have used the lazy pack generator (which I love BTW) for windows from version 2.4.1 for version 1.2.0 of TOTK. When using the "Add Black-screen fix (Xbox UI)" to match my game controller, gloom looks very strange, like in the screenshot below.

<a href="https://ibb.co/hHL6qjT"><img src="https://i.ibb.co/1K7gS54/gloom.jpg" alt="gloom" border="0" /></a>

When I regenerate the mod using "Remove Black-screen fix (UI Compatible)" the issue goes away. I have no other mods.

Using RTX 4090 / 7800X3D / Windows 11 64bit.

So in summary, this generated mod directory, gloom is fine.

_"Custom Lazy Pack - 3840x2160 - 60 FPS - Remove Black-screen fix (UI Compatible)"_

But this one is not, and shows the issue.

_"Custom Lazy Pack - 3840x2160 - 60 FPS - Add Black-screen fix (Xbox UI)"_

**Comment by SpockBauru** [comment]
Happens to me when I restart the game multiple times without closing Yuzu. 
Just for test, try to set the problematic pack, restart your whole PC and them start the game. In this order.

**Comment by mark007** [comment]
Thanks @SpockBauru but that didn't have any impact in this case. Only switching off the black screen fix version of the generated directory, fixes it for me, 100% reproducible.

**Comment by hoverbike1** [comment]
it seems there are multiple states of broken / working, and even some in between

here it is working as intended
https://cdn.discordapp.com/attachments/1107312895643095082/1130624082577215579/image.png

here it is trying its best
https://cdn.discordapp.com/attachments/1107312895643095082/1130629267953299487/image.png

and finally, here it has given up
https://cdn.discordapp.com/attachments/1107312895643095082/1130630012861681694/image.png

its hard to say whats causing it, as causing the issue is sporadic, i had to restart the game two dozen times to get these images.
whats more, i was able to cause the issues in while using the PS UI as well.

i guess i could give it a shot without the UI mods, but i don't think thats the issue.
this seems similar to an issue that was fixed over a month ago by the yuzu devs, weird to see it coming back, if only sporadically.

**Comment by mark007** [comment]
Interesting, and on my screenshot it looks even more different. It looks like some image/texture repeating/stretched. That's why I had suspected it was some UI mod / causing some sort of texture or image corruption of some sort.

**Comment by hoverbike1** [comment]
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/57139770/c910d03c-bf0e-4c2d-b9cb-c91f752d1355)
took me a while, but look what i was able to manage with the blackscreen-fix removed option.

if i were to take a shot in the woods, i'd blame either vram, but that seems unlikely in this case, excessive shadow resolution, which shouldnt be the case, especially since it works 'most of the time', i think this is likely just the return of an emulator issue that the yuzu devs thought they'd taken care of last month;

"Mainline 1444:
 - Fix the core cause of the gloom issue that would occur on The Legend of Zelda Tears of the Kingdom. Gloom rendering has been fixed for all system condition."

**Comment by mark007** [comment]
Ah ok. Feel free to close this one so, thanks. It must not be specific to the mods I guess. Unless you want to leave it open as a placeholder for any further checks.

**Comment by mark007** [comment]
Oh great. Looks to have fixed it for me! Nice find. Have not seen the weird gloom yet since using 4096.

**Comment by mark007** [comment]
I see this is fixed in 2.5.0, thanks a lot! Closing.
